Recrutify/Recrutify.Services/Validators/CreateCandidateValidator.cs

```diff
@@ -49,9 +62,40 @@ public CreateCandidateValidator()
                 .NotNull()
                 .NotEmpty();
             RuleForEach(c => c.BestTimeToConnect)
-                .NotEmpty();
+                .NotEmpty()
+                .Must(b => b >= 9 && b <= 18);
             RuleFor(c => c.PrimarySkillId)
+                .NotEmpty()
+                .MustAsync(CheckPrimarySkillsAsync);
+            RuleFor(c => c.CurrentJob)
+                .NotNull()
+                .NotEmpty()
+                .MaximumLength(50);
+            RuleFor(c => c.Certificates)
+               .NotNull()
+               .NotEmpty()
+               .MaximumLength(100);
+            RuleFor(c => c.AdditionalQuestions)
+               .NotNull()
+               .NotEmpty()
+               .MaximumLength(100);
+            RuleFor(c => c.AdditionalInfo)
+               .NotNull()
+               .NotEmpty()
+               .MaximumLength(100);
+            RuleFor(c => c.EnglishLevel)
+                .IsInEnum()
+                .NotEmpty();
+            RuleFor(c => c.ProjectLanguage)
+                .IsInEnum()
                 .NotEmpty();
+            RuleFor(c => c.GoingToExadel)
+                .NotEmpty();
```

Comment:
What you mean when write that bool should not be empty ?

Reply:
done